proposals/gc/MVP.md

```diff
@@ -354,7 +431,8 @@ This extends the [encodings](https://github.com/WebAssembly/function-references/
 | -0x14  | `(ref null ht)` | `ht : heaptype (s33)` | from funcref proposal |
 | -0x15  | `(ref ht)`      | `ht : heaptype (s33)` | from funcref proposal |
 | -0x16  | `i31ref`        |            | |
-| -0x17  | `(rtt n ht)`    | `n : u32`, `ht : heaptype (s33)` | |
+| -0x17  | `(rtt n $t)`    | `n : u32`, `$t : typeidx` | |
```

Comment:
Since you're here: the "Heap Types" section right below has a typo in the `i31` row (line 448), where it currently says -0x17. It should be consistent with line 433 and say -0x16.

Reply:
Thanks, fixed.